scripts/update-cocoa.sh

```diff
@@ -0,0 +1,27 @@
+#!/bin/bash
+
+cd $(dirname "$0")/../
+file='buildSrc/src/main/java/Config.kt'
```

Comment:
https://github.com/getsentry/sentry-kotlin-multiplatform/blob/ea9b2a45c737ddb46b1076114739a62405de7218/sentry-kotlin-multiplatform/sentry_kotlin_multiplatform.podspec#L15

Is the `podspec` updated from the `Config.kt`? Or do we need to update it too?

Reply:
Good catch, we need to update it as well